index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] [OPTIONAL] You'll need to import your stylesheet in order to make sure that your styles from your style.css work

## EXAMPLE
`<link rel="stylesheet" href="style.css">
`